scripts/build/build-stage.ps1

```diff
@@ -19,6 +19,7 @@ $Projects = @(
     "Microsoft.DotNet.Tools.New",
     "Microsoft.DotNet.Tools.Pack",
     "Microsoft.DotNet.Tools.Publish",
+    "dotnet-restore",
```

Comment:
I really like this folder name, I think we should rename all of them (separately). @davidfowl do you know if there are any problems with using dash in project folder name? (i.e. namespace in new files in VS)